### PR TITLE
set up github actions

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

File: .github/workflows/TagBot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: TagBot
+on:
+  schedule:
+    - cron: 0 0 * * *
+jobs:
+  TagBot:
```
Comment (dkarrasch) [inline .github/workflows/TagBot.yml:6]:
```suggestion
name: TagBot
on:
  issue_comment:
    types:
      - created
  workflow_dispatch:
jobs:
  TagBot:
    if: github.event_name == 'workflow_dispatch' || github.actor == 'JuliaTagBot'
```
That's the latest recommendation, to have it run only on specific triggers, and not daily.

File: README.md [hunk 2]
```diff
@@ -70,19 +70,14 @@ Both the packages below provide a convenience macro to provide einstein summatio
...
 
-[docs-latest-img]: https://img.shields.io/badge/docs-latest-blue.svg
-[docs-latest-url]: https://kristofferc.github.io/Tensors.jl/latest/
 
 [docs-stable-img]: https://img.shields.io/badge/docs-stable-blue.svg
 [docs-stable-url]: https://kristofferc.github.io/Tensors.jl/stable
 
-[travis-img]: https://travis-ci.org/KristofferC/Tensors.jl.svg?branch=master
-[travis-url]: https://travis-ci.org/KristofferC/Tensors.jl
+[ci-img]: https://github.com/Ferrite-FEM/Tensors.jl/workflows/CI/badge.svg
```
Comment (dkarrasch) [inline README.md:77]:
```suggestion
[ci-img]: https://github.com/Ferrite-FEM/Tensors.jl/workflows/CI/badge.svg?branch=master
```

File: README.md [hunk 3]
```diff
@@ -70,19 +70,14 @@ Both the packages below provide a convenience macro to provide einstein summatio
...
+[ci-url]: https://github.com/Ferrite-FEM/Tensors.jl/actions?query=workflows/CI
```
Comment (dkarrasch) [inline README.md:78]:
```suggestion
[ci-url]: https://github.com/Ferrite-FEM/Tensors.jl/actions?query=workflows%3ACI+branch%3Amaster
```

File: README.md [hunk 4]
```diff
@@ -70,19 +70,14 @@ Both the packages below provide a convenience macro to provide einstein summatio
...
 
-[appveyor-img]: https://ci.appveyor.com/api/projects/status/xe0ghtyas12wv555/branch/master?svg=true
-[appveyor-url]: https://ci.appveyor.com/project/KristofferC/Tensors-jl/branch/master
+[issues-url]: https://github.com/Ferrite-FEM/Tensors.jl/issues
 
-[issues-url]: https://github.com/KristofferC/Tensors.jl/issues
-
-[codecov-img]: https://codecov.io/gh/KristofferC/Tensors.jl/branch/master/graph/badge.svg
-[codecov-url]: https://codecov.io/gh/KristofferC/Tensors.jl
+[codecov-img]: https://codecov.io/gh/Ferrite-FEM/Tensors.jl/branch/master/graph/badge.svg
```
Comment (dkarrasch) [inline README.md:82]:
```suggestion
[codecov-img]: https://codecov.io/gh/Ferrite-FEM/Tensors.jl/branch/master/graph/badge.svg?branch=master
```

File: README.md [hunk 5]
```diff
@@ -70,19 +70,14 @@ Both the packages below provide a convenience macro to provide einstein summatio
...
+[codecov-url]: https://codecov.io/gh/Ferrite-FEM/Tensors.jl
```
Comment (dkarrasch) [inline README.md:83]:
```suggestion
[codecov-url]: https://codecov.io/gh/Ferrite-FEM/Tensors.jl?branch=master
```

**Review by dkarrasch — COMMENTED** [review]
A few minor changes that make the badges refer to the master branch only. Maybe unnecessary, but that's how I found many people do it.

File: README.md (hunk 2 above)
Comment (KristofferC) [reply README.md:77]: Is master not the default? What is it showing otherwise?

Comment (dkarrasch) [reply README.md:77]: Somebody told me that it might interfere with the most recent commit to some PR, but I don't know.

**Review by fredrikekre — APPROVED** [review]

**Pull request merged**